doc/api/readline.md

```diff
@@ -722,6 +723,123 @@ const { createInterface } = require('readline');
 })();
 ```
 
+## TTY keybindings
```

Comment:
I feel we should also add the key bindings for reverse search that we have recently added to the repl (Refs: https://github.com/nodejs/node/pull/31006). @BridgeAR thoughts?

Reply:
@HarshithaKP would you please also add a reference to the mentioned line? :-)

Reply:
@BridgeAR, thanks. Fixed it.